test/unit/middlewares/application-validator.test.ts

```diff
@@ -61,10 +61,10 @@ describe("application validator test", function () {
     });
 
     it("should not call the next function if number of hours is not a number", async function () {
-       const rawData = {
-         ...applicationsData[6],
-         numberOfHours: '10'
-       };
+      const rawData = {
+        ...applicationsData[6],
+        numberOfHours: "10",
+      };
```

Comment:
Here also

Reply:
They were done by lint in my vscode system